invenio_rdm_records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -24,46 +24,53 @@
     "access_right": {
       "default": "open",
       "description": "Access right for record.",
-      "enum": [
-        "open",
-        "embargoed",
-        "restricted",
-        "closed"
-      ],
-      "type": "string"
-    },
-    "recid": {
-      "description": "Invenio record identifier (integer).",
-      "type": "number"
-    },
-    "title": {
-      "description": "Record title.",
       "type": "string"
     },
-    "description": {
-      "description": "Description/abstract for record.",
-      "type": "string"
-    },
-    "owners": {
-      "description": "List of user IDs that are owners of the record.",
+    "additional_descriptions": {
```

Comment:
It's because of the "convenient search constraint" coupled with the way we index documents:

1- convenient search: it is apparently known (or at least Zenodo and others want) that `description=mySearchTerm` can be entered in the search box to search on description. If a complex object is in that field that can't be done because...
2- naive indexing: as much as possible we naively index records from their jsonschema to their ElasticSearch mapping.

That said, this is "fixable" to achieve storing descriptions/titles in a single field and yet have convenient search keywords: building custom indexing hook for those, having a query parser module to map a passed query into our internal representation... We won't be getting to those by the end of the month so we will tackle it later. IMHO, the custom indexing hook seems like the simplest approach to achieve this. New issue for this: #24